# file: C:\Users\micha\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\_pytest\pathlib.py
# hypothesis_version: 6.138.15

[420, 448, 1921, '.', '.lock', '.pyc', '.pyo', '1', '\\\\', '\\\\?\\', '\\\\?\\UNC\\', '_', '_AnyPurePath', '__init__', '__init__.py', '__module__', '__path__', 'append', 'current', 'errno', 'garbage-*', 'importlib', 'prepend', 'win', 'win32', 'winerror']